test/gtest/ucp/test_ucp_proto_mock.cc

```diff
@@ -662,3 +653,48 @@ UCS_TEST_P(test_ucp_proto_mock_self, rkey_ptr)
 }
 
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_proto_mock_self, self, "self")
+
+class test_ucp_proto_mock_gpu : public test_ucp_proto_mock {
+public:
+    test_ucp_proto_mock_gpu()
+    {
+        mock_transport("rc_mlx5");
+    }
+
+    virtual void init() override
+    {
+        add_mock_iface("mock", [](uct_iface_attr_t &iface_attr) {
+            iface_attr.cap.am.max_short  = 2000;
+            iface_attr.bandwidth.shared  = 28e9;
+            iface_attr.latency.c         = 600e-9;
+            iface_attr.latency.m         = 1e-9;
+            iface_attr.cap.get.max_zcopy = 16384;
+        });
+        test_ucp_proto_mock::init();
+    }
+};
+
+UCS_TEST_P(test_ucp_proto_mock_gpu, cuda_managed_ppln_host_frag,
+           "RNDV_FRAG_MEM_TYPES=host", "IB_NUM_PATHS?=1", "MAX_RNDV_LANES=1")
+{
+    send_recv_am(1024, UCS_MEMORY_TYPE_CUDA_MANAGED);
+
+    ucp_proto_select_key_t key = any_key();
+    key.param.op_id_flags      = UCP_OP_ID_AM_SEND;
+    key.param.op_attr          = 0;
+    key.param.mem_type         = UCS_MEMORY_TYPE_CUDA_MANAGED;
+
+    check_ep_config(sender(), {
+        {0, 0,    "short",   "rc_mlx5/mock"},
+        {1, 8246, "copy-in", "rc_mlx5/mock"},
+        {8247, 512 * UCS_KBYTE,
+         "rendezvous cuda_copy, fenced write to remote, frag host, cuda_copy, frag host",
+         "rc_mlx5/mock"},
+        {(512 * UCS_KBYTE) + 1, INF,
+         "rendezvous pipeline cuda_copy, fenced write to remote, frag host, cuda_copy, frag host",
+         "rc_mlx5/mock"},
+        }, key);
+}
+
+UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_proto_mock_gpu, rcx_gpu,
```

Comment:
use `UCP_INSTANTIATE_TEST_CASE_TLS_GPU_AWARE` instead?
(maybe also update it to include gpu ipc tks as well)

Reply:
UCP_INSTANTIATE_TEST_CASE_TLS_GPU_AWARE does not include IPC on purpose, probably better not change it to not affect other tests. UCP_INSTANTIATE_TEST_CASE_GPU_AWARE macro has specific case with IPC.

Reply:
ok